Models/rotors/fenestronanimation.xml

```diff
@@ -25,4 +25,43 @@
     <inherits-from>Effects/model-transparent</inherits-from>
     <object-name>rotor_disc_T</object-name>
   </effect>
+
+  <animation>
+    <!-- Rotate the fenestron blades -->
+    <type>rotate</type>
+    <object-name>Tblade</object-name>
+    <property>rotors/tail/blade/position-deg</property>
+    <factor>1</factor>
+    <center>
+      <x-m>0</x-m>
+      <y-m>0</y-m>
+      <z-m>0</z-m>
+    </center>
+    <axis>
+      <x>0</x>
+      <y>1</y>
+      <z>0</z>
+    </axis>
+  </animation>
+  <animation>
+    <!-- Hide the fenestron blades when rotating with high rpm -->
+    <type>select</type>
```

Comment:
Even for the Main rotor it wouldn't have been necessary, but I found it easier to animate this way.
Usually I prefer less submodels as possible, since they decrease perfomance and fps. 

The original code looked like that, the blades had been joined into one model with the option to be separated and animated separated with help of the fgfs animation script in Blender 2.49:
`
<!--tail rotor animation-->
	<animation>
		<type>material</type>
		<object-name>rotor_disc_T</object-name>
		<transparency>
			<alpha-prop>rotors/tail/rpm</alpha-prop>
			<factor>0.0015</factor>
			<offset>-0.2</offset>
			<max>0.55</max>
		</transparency>
		<threshold>0.001</threshold>
	</animation>

	<animation>
		<type>noshadow</type>
		<object-name>rotor_disc_T</object-name>
	</animation>

	<animation>
		<type>material</type>
		<object-name>Tblade</object-name>
		<!--<object-name>tblade1</object-name>
    <object-name>tblade2</object-name>
    <object-name>tblade3</object-name>
    <object-name>tblade4</object-name>
    <object-name>tblade5</object-name>
    <object-name>tblade6</object-name>
    <object-name>tblade7</object-name>
    <object-name>tblade8</object-name>
    <object-name>tblade9</object-name>
    <object-name>tblade10</object-name>-->
		<global type="bool">true</global>
		<transparency>
			<alpha-prop>rotors/tail/rpm</alpha-prop>
			<factor>-0.005</factor>
			<offset>1</offset>
			<max>1</max>
		</transparency>
	</animation>



	<animation>
		<type>rotate</type>
		<object-name>Tblade</object-name>
		<!--<object-name>tblade2</object-name>
		<object-name>tblade2</object-name>
    <object-name>tblade3</object-name>
		<object-name>tblade4</object-name>
    <object-name>tblade5</object-name>
		<object-name>tblade6</object-name>
    <object-name>tblade7</object-name>
		<object-name>tblade8</object-name>
    <object-name>tblade9</object-name>
		<object-name>tblade10</object-name>-->
		<property>rotors/tail/blade[0]/position-deg</property>
		<axis>
			<x1-m>5.106</x1-m>
			<y1-m>-0.217</y1-m>
			<z1-m>2.476</z1-m>
			<x2-m>5.106</x2-m>
			<y2-m>0.217</y2-m>
			<z2-m>2.476</z2-m>
		</axis>
	</animation>

	<animation>
		<name>tailrotors</name>
		<object-name>Tblade</object-name>
		<!--<object-name>tblade1</object-name>
		<object-name>tblade2</object-name>
    <object-name>tblade3</object-name>
		<object-name>tblade4</object-name>
    <object-name>tblade5</object-name>
		<object-name>tblade6</object-name>
    <object-name>tblade7</object-name>
		<object-name>tblade8</object-name>
    <object-name>tblade9</object-name>
		<object-name>tblade10</object-name>-->
	</animation>

	<animation>
		<type>select</type>
		<object-name>rotor_disc_T</object-name>
		<condition>
			<greater-than>
				<property>/rotors/tail/rpm</property>
				<value>200</value>
			</greater-than>
		</condition>
	</animation>

	<animation>
		<type>select</type>
		<object-name>tailrotors</object-name>
		<condition>
			<less-than>
				<property>/rotors/tail/rpm</property>
				<value>201</value>
			</less-than>
		</condition>
	</animation>
`
When we want to animate each blade, then the solution of the [EC130 B4](https://gitlab.com/mhab/ec130/tree/master/Models/TailRotor) is easier to implement.